package.json

```diff
@@ -38,7 +38,7 @@
     "tap": "^14.10.8"
   },
   "engines": {
-    "node": ">=10.10.0"
+    "node": ">=10.12.0"
```

Comment:
10 is EOL, just drop it entirely and only support 12+ since we already need to do a major?

Reply:
I'm going to go ahead and land this as a breaking change, I'm fine with the requirement for Node `10.12`, and it means my team can keep using `c8` until we drop Node 10 (hopefully in the next couple months).